src/MagnumPlugins/TinyGltfImporter/Test/TinyGltfImporterTest.cpp

```diff
@@ -1945,7 +1945,7 @@ void TinyGltfImporterTest::texture() {
     CORRADE_COMPARE(texture->type(), TextureData::Type::Texture2D);
 
     CORRADE_COMPARE(texture->magnificationFilter(), SamplerFilter::Linear);
-    CORRADE_COMPARE(texture->minificationFilter(), SamplerFilter::Nearest);
+    CORRADE_COMPARE(texture->minificationFilter(), SamplerFilter::Linear);
```

Comment:
A more explicit test for this would be nice (testing all defaults as mentioned in the plugin docs).

Reply:
There is `textureDefaultSampler already` 🤔

Reply:
Oh. Okay then (sorry, should have checked better).